package.json

```diff
@@ -56,7 +56,14 @@
     "rimraf": "^2.5.3"
   },
   "eslintConfig": {
-    "extends": "eslint-config-sinon"
+    "extends": "eslint-config-sinon",
+    "globals": {
+      "ArrayBuffer": false,
+      "Map": false,
```

Comment:
I only added the globals that are currently being used in the source. We can always add more